collector/bgp.go

```diff
@@ -48,7 +48,7 @@ func getBGPDesc() map[string]*prometheus.Desc {
 
 	if *bgpPeerDescs {
 		bgpPeerLabels = append(bgpLabels, "peer", "peer_as", "peer_desc")
-}
+	}
```

Comment:
I noticed this change which does not belong in this PR. So I created a separate one for the formatting: https://github.com/tynany/frr_exporter/pull/51

Reply:
Thanks for merging #51. Can you rebase this PR on master?